spec/linter-gotype-spec.js

```diff
@@ -0,0 +1,37 @@
+'use babel';
+
+import * as path from 'path';
+import {
+  // eslint-disable-next-line no-unused-vars
+  it, fit, wait, beforeEach, afterEach,
+} from 'jasmine-fix';
+
+const { lint } = require('../lib/linter-gotype.js').provideLinter();
```

Comment:
```suggestion
const { lint } = require('../lib/init').provideLinter();
```
See comment in `package.json` for reasoning.